Xamarin.Forms.Platform.Android/Renderers/TableViewModelRenderer.cs

```diff
@@ -13,13 +16,20 @@ public class TableViewModelRenderer : CellAdapter
 		protected readonly Context Context;
 		ITableViewController Controller => _view;
 		Cell _restoreFocus;
+		Cell[] cellCache;
```

Comment:
Please be consistent with the style of the file. The other fields are prefixed with an underscore.